programs/clearing_house/src/controller/position.rs

```diff
@@ -175,13 +197,6 @@ pub fn update_position_and_market(
         }
     };
 
```

Comment:
note: removed net_baa update in this fcn so net_baa only updates in update_user_and_market_position - this is ok bc  update_position_and_market was used in - liquidation (liq’r gets the position from the liq’d) &  fulfill_order_with_match (taker & maker) where the net_baa wouldnt change anyway

Reply:
why did you want to do that? not sure i love the decision, because it means we need to remember we always have to call update_position_and_market twice

Reply:
so we have two fcns 
- `update_user_and_market_position`: which updates the market's stats (and not position) and the user's position 
- `update_position_and_market`: which updates the users position and updates the markets position (including stats) 

when we give a slice to the lps and the market we need to do it through update_position_and_market -- however, update_user_and_market_position calls update_position_and_market and so when update_user_and_market_position updates net_baa by the full delta we get a double call of the net_baa stat